Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/EditorBaseComponentMode.cpp

```diff
@@ -53,13 +53,34 @@ namespace AzToolsFramework
             ComponentModeViewportUiRequestBus::Event(
                 GetComponentType(), &ComponentModeViewportUiRequestBus::Events::RegisterViewportElementGroup,
                 GetEntityComponentIdPair(), elementIdsToDisplay);
-            // create the component mode border with the specific name for this component mode
-            ViewportUi::ViewportUiRequestBus::Event(
-                ViewportUi::DefaultViewportId, &ViewportUi::ViewportUiRequestBus::Events::CreateViewportBorder, GetComponentModeName(),
-                []
-                {
-                    ComponentModeSystemRequestBus::Broadcast(&ComponentModeSystemRequests::EndComponentMode);
-                });
+
+            bool borderVisible;
```

Comment:
update: Please always initialize bus result variables otherwise we may have an uninitialized variable read if the bus isn't connected or isn't found. So this would be `bool borderVisible = false`.